SECURITY.md

```diff
@@ -0,0 +1 @@
+If you have discovered a security vulnerability, please email tech-security@zalando.de
```

Comment:
security isn't our headache 😆

Reply:
This is the standard security.md file recommended by rules of play :)